src/plugins/unbounce-event-mixin.js

```diff
@@ -7,6 +7,9 @@ export default {
 		const unbounceRoute = this.$route?.meta?.unbounce ?? false;
```

Comment:
Can we convert this to mounted So it doesn't try to run it on the server?

Reply:
Yes I think thats fine 👍